flow/project.py

```diff
@@ -616,10 +767,15 @@ class FlowGroupEntry(object):
         commands to execute.
     :type options:
         str
+    :param aggregator:
+        Aggregation parameters :py:class:`FlowGroup` to be created.
+    :type aggregator:
+        tuple
     """
-    def __init__(self, name, options=""):
+    def __init__(self, name, options="", flow_aggregate=(None, None)):
```

Comment:
What parameters here? Since we do not have an actual `Aggregate` object that knows how do do all of the aggregation and sorting, what gets passed here? In addition, it is not clear in the documentation or the code here what is represented by the members of the tuple.

I recommend we create an `Aggregate` class that after jobs have been filtered, yields the appropriate aggregates.